### ISSUE TITLE
App v 3.132.2 does not load (white screen lockup) on De-Googled Android

### ISSUE THREAD

**Issue by yralike** [standardnotes/forum#2380]
Version 3.132.2 does not load (white screen lockup) on De-Googled Android devices.

**To Reproduce**
1. Launch app.
2. See error

**Expected behavior**
App to launch as normal.

**Screenshots**
N/A White screen of lockup.

**Smartphone (please complete the following information):**
 - Android, De-Googled.

**Comment by effieeee** [comment]
Hi @yralike, is this still happening on the latest app versions, and have you also tried [clearing the app data](https://docs.standardnotes.com/troubleshooting/reset-apps/#android) for Standard Notes on your phone, then relaunching the app?

**Comment by yralike** [comment]
Yes this is still happening with the latest app version.  And having cleared app data, and relaunching the app.  I have also completely removed the app and data and reinstalled fresh and the latest version is inoperable with complete white screen of lock up.  All prior versions have worked fine.

**Comment by Dodger-nzl** [comment]
I was having the same issue on my Android phone. 
Clearing the app cache and storage fixed the issue, even after an uninstall\reinstall didn't .  
Thanks.

**Comment by theAkito** [comment]
I just installed the app. Happened on my mobile phone with a custom ROM, which has minimal Google dependencies on it.

Only thing I can do, is exiting the app, by going back and confirming the dialogue.
